<ui version="4.0" >
 <class>Viewer</class>
 <widget class="QMainWindow" name="Viewer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Visocyte</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="Icons/icons.qrc" >:/Icons/help.png</iconset>
  </property>
  <property name="iconSize" >
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>

  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" name="verticalLayout">

    <item>
     <widget class="QVTKOpenGLNativeWidget" name="qvtkWidget" native="true">
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>

    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">

      <item>
        <widget class="QToolBar" name="toolBar" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="iconSize" >
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <attribute name="toolBarArea" >
          <enum>TopToolBarArea</enum>
         </attribute>
        </widget>
      </item>

      <item>
        <widget class="QSlider" name="progressSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
        </widget>
      </item>

     </layout>
    </item>

   </layout>
  </widget>

  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpenFile" />
    <addaction name="actionSave" />
    <addaction name="separator" />
    <addaction name="actionHelp" />
    <addaction name="actionExit" />
   </widget>
   <addaction name="menuFile" />
  </widget>


        <action name="actionOpenFile" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="icon" >
          <iconset resource="Icons/icons.qrc" >:/Icons/fileopen.png</iconset>
         </property>
         <property name="text" >
          <string>Open File...</string>
         </property>
        </action>
        <action name="actionExit" >
         <property name="text" >
          <string>Exit</string>
         </property>
        </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLNativeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons/icons.qrc" />
 </resources>
 <connections/>
</ui>
